### PR TITLE
Make

### PR DESCRIPTION
**Pull request by onishiweb**
Adds a Makefile to make running development tasks easier locally.

Also, updates readme to conform to a consistent standard (comparing to the [Origami Build Service](https://github.com/Financial-Times/origami-build-service)).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +86,20 @@ In dev, these are configured in the `.env` file.  In live, it's `heroku config`
...
 * `VIEW_CACHE_PATH`: Path on disk to use to cache view templates in Twig
 * `DEBUG_KEY`: String, if set in a `Debug` HTTP header, will set dev mode to true for that request only.
+
+### Orchestration files
+
+The following files are used in build, test and deploy automation:
+
+* `.dockerignore`: used to ignore things when adding files to the Docker image.  Generally this will be the same as the `.gitignore` file as the build happens at the container creation time.  See the `Dockerfile` for more info.
+* `app.json`: TODO
+* `Dockerfile`: TODO
```
Comment (AlbertoElias) [inline README.md:106]: Why not fill these in if you're adding the section

Comment (AlbertoElias) [inline README.md:106]: oh, just noticed it was moved, sorry!

Comment (onishiweb) [inline README.md:106]: Ah yeh, I'm going to look into filling those in, but they were done before my time so I'm not entirely sure how to explain them. Might need some help with that.

**Comment by AlbertoElias** [comment]
lgtm

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+deploy: build
+	@git describe --tags > ./appversion
+	@heroku docker:release --app origami-registry-eu
+	@rm -f ./appversion
+
+build:
+	@if [[ "$$(docker-machine ls | grep dev)" == *"Stopped"* ]]; \
```
Comment (rowanmanning) [inline Makefile]: Could this be a one-line conditional that runs `exit 0` if the machine is not stopped? Then you wouldn't need all the escaped line breaks

**Comment by rowanmanning** [comment]
Looks good 👍 just one question

File: Makefile (hunk 2 above)
Comment (onishiweb) [inline Makefile]: Yep it definitely could be. Will just update that to test for `Running` instead

Comment (onishiweb) [inline Makefile]: I've tried this and can't seem to get Make to exit the task without throwing an error and finding it hard to find any good documentation on how to exit a task without make throwing an error. This wouldn't be a big issue but it's a dependency for the build task so I'm not sure how to exit the dependency and allow the other task to complete.

**Comment by rowanmanning** [comment]
Looks good to me 👍

**Pull request merged**